PATCHES.md

```diff
@@ -1,20 +1,28 @@
 I use [Hopper Disassembler](http://www.hopperapp.com), it's available for OS X and Linux. You can use trial version that's is limited to 30 minutes which is enough.
 
+You can use Hopper to find the patches for all versions of DF. So you can run Hopper on OSX or Linux and use it to disassemble all of OSX, Windows and Linux versions.
+
 Note: usually addresses don't change much, and usually they increase. So if some address you found is completely different from the address in previous version, better check again if it's correct.
 
+The first time you do the patching on a new version it's recommended to first follow the instructions for a previous version that is already completed. That means you already have the found addresses which you can use to verify that you are understanding and following the instructions correctly. You can then refer back to the known-good version when searching in the new version, which can be very helpful when the new version has changed enough that it doesn't quite match the patterns described below.
+
 1. To find the address for `A_LOAD_MULTI_PDIM`:
 
 	**On Windows** find a function that references string "`Tileset not found`".
 	**On OS X** find a function that calls `IMG_Load` in its very beginning, and next `SDL_SetAlpha`, `SDL_CreateRGBSurface`, and `SDL_SetAlpha` again.
 	**On Linux** not required.
 
-2. Go to the address of vtable for `viewscreen_dwarfmodest` (from symbols.xml), you need *third* DWORD at that address, it's the address of viewscreen's `render()` method. Go to that function.
+2. Go to the address of vtable for `viewscreen_dwarfmodest` (from symbols.xml), you need *third* DWORD at that address, it's the address of viewscreen's `render()` method. Go to that function - note that on Windows, you will jump into the middle of a function, not the beginning.
```

Comment:
Oh OK!  So is it the case that 00641E40 in fact is the start of a function?  Just Hopper doesn't realise it?  I can't tell from your screenshot whether 00641E40 is the start of a func or not, but that would make sense given the instructions.

In that case I will remove that comment I added to patches.md.  Or rather I will update it to say that it might not look like the start of a function in Hopper but just follow the instructions regardless.

PS I'm nearly done with Linux, will push in 20 mins or so I hope once it's tested.